collectors/nvd/collectors.py

```diff
@@ -56,32 +58,68 @@ def response2result(self, vulnerabilities: list) -> list:
         filtering out everything unnecessary and simplifying
         """
 
-        def compose_cvss(metrics: list) -> Union[str, None]:
+        def get_description(descriptions: list) -> Union[str, None]:
             """
-            compose the result vector helper
+            Return a description only if it is written in English.
             """
-            for metric in metrics:
-                # we only care for NVD record
-                if metric.source == "nvd@nist.gov":
-                    return f"{metric.cvssData.baseScore}/{metric.cvssData.vectorString}"
+            return [d.value for d in descriptions if d.lang == "en"][0] or None
```

Comment:
Please match the language using the [official spec regex](https://github.com/CVEProject/cve-schema/blob/d2b6b34a527d18021209745d9ae19d966cc212f6/schema/v5.0/CVE_JSON_5.0_schema.json#L1030). Moreover, the spec requires that an English description is provided: 

So something like this:

```suggestion
            # Find the first English description
            # Ref: https://github.com/CVEProject/cve-schema/blob/d2b6b34a527d18021209745d9ae19d966cc212f6/schema/v5.0/CVE_JSON_5.0_schema.json#L744
            for d in descriptions:
                if re.match("^en([_-][A-Za-z]{4})?([_-]([A-Za-z]{2}|[0-9]{3}))?$", d.lang):
                    return d.value
```

Reply:
That regex is official only for the `cve-schema` project. And the result in nvd can look like this: 
```
'descriptions': [
    {'lang': 'en', 'value': 'The ip6_find_1stfragopt function in net/ipv6/output_core.c ...'}, 
    {'lang': 'es', 'value': 'La función ip6_find_1stfragopt en el archivo net/ipv6/output_core.c ...'}
]
```

So I think the regex will not ease the work here.

Reply:
Aha, sorry about the confusion!

I find it strange that they decided to use a different format for nvdlib, but you are correct. Tried with `CVE-2022-38014` which has `en-US` `descriptions` in `cvelistV5` and that CVE has two `descriptions` in `nvdlib`: `en` and `es`, no trace of `en-US`.